.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -0,0 +1,53 @@
+name: Bug Report
+description: Something isn't working as expected
+labels: [bug]
+body:
+  - type: input
+    id: version
+    attributes:
+      label: Swift version
+      description: Which version of Swift are you using? If you are unsure, insert the output of `path/to/swift --version`
+      placeholder: Eg. swiftlang-5.10.0.13, swift-DEVELOPMENT-SNAPSHOT-2024-05-01-a
+  - type: input
+    id: platform
+    attributes:
+      label: Platform
+      description: What operating system are you seeing the issue on?
+      placeholder: Eg. Ubuntu 22.04, Windows 11, macOS 14
+  - type: input
+    id: editor
+    attributes:
+      label: Editor
+      description: Which text editor are you using?
+      placeholder: Eg. Visual Studio Code with Swift plugin 1.9.0, neovim
```

Comment:
```suggestion
      description: Which text editor are you using (and LSP extension/plugin if applicable)?
      placeholder: Eg. Visual Studio Code with Swift extension 1.9.0, Neovim, etc.
```

Reply:
I think `etc.` isn’t necessary because we already have `E.g.` at the start of the placeholder.

Reply:
Was more the extension/plugin. I don't mind if you add the etc or not :)